CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# 8.0.0 - [#125](https://github.com/ServiceInnovationLab/openfisca-aotearoa/pull/125)
```

Comment:
I'm not sure we need a full version leap for this change.

Reply:
Contributing guide says:
Major bump:    Renaming or removing a variable from the tax and benefit system.